<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>836</width>
    <height>835</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>9</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#Form
{
border-image: url(:/Icon/icons/solar2_edited.jpg)0 0 0 0 stretch stretch;
}</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>220</y>
     <width>671</width>
     <height>581</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QFrame
{
background:rgba(0,0,0,0.8);
border-radius:15px
}</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QPushButton" name="SetButton">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>490</y>
      <width>271</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
font-family:century gothic;
font-size:24px;
background:orange;
color:white;
border-radius:15px;
}
QPushButton:hover:!pressed
{
background:white;
color:orange;
border-radius:15px;
}</string>
    </property>
    <property name="text">
     <string>Set</string>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_Name">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>130</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-family:century gothic;
font-size:24px;
background:transparent;
border:none;
color:#717072;
border-bottom:1px solid #717072</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string>Power System</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_Key">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>130</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">	font-family:century gothic;
	font-size:24px;
background:transparent;
border:none;
color:#717072;
border-bottom:1px solid #717072</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Normal</enum>
    </property>
    <property name="dragEnabled">
     <bool>false</bool>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
    <property name="placeholderText">
     <string>Key</string>
    </property>
    <property name="cursorMoveStyle">
     <enum>Qt::LogicalMoveStyle</enum>
    </property>
    <property name="clearButtonEnabled">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>60</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Century Gothic</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background:transparent;
color:white;


</string>
    </property>
    <property name="text">
     <string>Setup</string>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="AddButton">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>140</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
font-family:century gothic;
font-size:24px;
background:orange;
color:white;
border-radius:15px;
}
QPushButton:hover:!pressed
{
background:white;
color:orange;
border-radius:15px;
}</string>
    </property>
    <property name="text">
     <string>Add</string>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QTableWidget" name="TableWidget">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>200</y>
      <width>271</width>
      <height>211</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">
QHeaderView::section { 
color:black; background-color: #FFC300; 
}
QTableCornerButton::section {
 background-color:#FFC300;
textcolor:#FFC300;
color:#FFC300;
 }
textcolor:#717072;
background-color:rgba(255,255,255,0.2);
font: 11pt &quot;Century Gothic&quot;;
color-text: 
</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="midLineWidth">
     <number>0</number>
    </property>
    <property name="autoScroll">
     <bool>true</bool>
    </property>
    <property name="tabKeyNavigation">
     <bool>true</bool>
    </property>
    <property name="showDropIndicator" stdset="0">
     <bool>true</bool>
    </property>
    <property name="alternatingRowColors">
     <bool>false</bool>
    </property>
    <property name="showGrid">
     <bool>true</bool>
    </property>
    <property name="gridStyle">
     <enum>Qt::SolidLine</enum>
    </property>
    <property name="sortingEnabled">
     <bool>true</bool>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
    <property name="cornerButtonEnabled">
     <bool>true</bool>
    </property>
    <property name="columnCount">
     <number>2</number>
    </property>
    <attribute name="horizontalHeaderVisible">
     <bool>true</bool>
    </attribute>
    <attribute name="horizontalHeaderCascadingSectionResizes">
     <bool>false</bool>
    </attribute>
    <attribute name="horizontalHeaderMinimumSectionSize">
     <number>0</number>
    </attribute>
    <attribute name="horizontalHeaderHighlightSections">
     <bool>true</bool>
    </attribute>
    <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
     <bool>true</bool>
    </attribute>
    <attribute name="horizontalHeaderStretchLastSection">
     <bool>true</bool>
    </attribute>
    <attribute name="verticalHeaderVisible">
     <bool>true</bool>
    </attribute>
    <attribute name="verticalHeaderCascadingSectionResizes">
     <bool>false</bool>
    </attribute>
    <row>
     <property name="text">
      <string>1</string>
     </property>
     <property name="font">
      <font>
       <pointsize>8</pointsize>
      </font>
     </property>
    </row>
    <column>
     <property name="text">
      <string>Name</string>
     </property>
     <property name="font">
      <font>
       <stylestrategy>PreferDefault</stylestrategy>
      </font>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Key</string>
     </property>
    </column>
   </widget>
   <widget class="QPushButton" name="LSButton">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>490</y>
      <width>271</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
font-family:century gothic;
font-size:24px;
background:orange;
color:white;
border-radius:15px;
}
QPushButton:hover:!pressed
{
background:white;
color:orange;
border-radius:15px;
}</string>
    </property>
    <property name="text">
     <string>Last Settings</string>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="ZigbeeButton">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>220</y>
      <width>231</width>
      <height>45</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
font-family:century gothic;
font-size:20px;
background:red;
color:white;
border-radius:15px;
}
QPushButton:hover:!pressed
{
background:white;
color:red;
border-radius:15px;
}</string>
    </property>
    <property name="text">
     <string>Zigbee Configuration</string>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="frame_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>841</width>
     <height>111</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
background:rgba(255,255,255,0.2);

</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>30</y>
      <width>551</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>century gothic</family>
      <pointsize>16</pointsize>
      <weight>50</weight>
      <italic>true</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background:transparent;
color:white;</string>
    </property>
    <property name="text">
     <string>Configuration Page</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
    <property name="openExternalLinks">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>30</y>
      <width>71</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:transparent;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="Config.qrc">:/Icon/icons/settings.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>30</y>
      <width>71</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:transparent;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="Config.qrc">:/Icon/icons/settings.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QToolButton" name="toolButton">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>140</y>
     <width>121</width>
     <height>121</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:orange;
border-radius:60px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="Config.qrc">
     <normaloff>:/Icon/icons/iconfinder_Construction_engineer_config_6588863.png</normaloff>:/Icon/icons/iconfinder_Construction_engineer_config_6588863.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>90</width>
     <height>90</height>
    </size>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="autoRepeat">
    <bool>false</bool>
   </property>
   <property name="autoExclusive">
    <bool>false</bool>
   </property>
   <property name="autoRaise">
    <bool>false</bool>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="Config.qrc"/>
 </resources>
 <connections/>
</ui>
